Clover/app/src/main/java/org/floens/chan/ui/controller/export/LoadingViewController.java

```diff
@@ -0,0 +1,50 @@
+package org.floens.chan.ui.controller.export;
+
+import android.app.Activity;
+import android.content.Context;
+import android.os.Build;
+import android.view.Window;
+
+import org.floens.chan.R;
+import org.floens.chan.controller.Controller;
+import org.floens.chan.utils.AndroidUtils;
+
+public class LoadingViewController extends Controller {
```

Comment:
Like 99% of this class code can be extracted to an abstract class and then other controllers can inherit from it.